graspologic/plot/plot.py

```diff
@@ -671,6 +673,229 @@ def pairplot(
+    font_scale : float, optional, default: 1
+        Separate scaling factor to independently scale the size of the font
+        elements.
+    alpha : float, optional, default: 0.7
+        Opacity value of plotter markers between 0 and 1
+    figsize : tuple
+        the size of the 2d subplots configuration
+    kde : boolean
+        allows for access of KDE argument in histplot
+
+    See Also
+    --------
+    graspologic.cluster.AutoGMMCluster
+    sklearn.mixture.GaussianMixture
+    """
+    # Handle X and labels
+    if labels is not None:
+        check_X_y(X, labels)
+    # Handle gmm
+    if gmm is None:
+        msg = "You must input a sklearn.mixture.GaussianMixture"
+        raise NameError(msg)
+    Y_, means, covariances = gmm.predict(X), gmm.means_, gmm.covariances_
+    data = pd.DataFrame(data=X)
+    n_components = gmm.n_components
+    # setting up the data DataFrame
+    if labels is None:
+        lab_names = [i for i in range(n_components)]
+        data["labels"] = np.asarray([lab_names[Y_[i]] for i in range(Y_.shape[0])])
+    else:
+        data["labels"] = labels
+    data["clusters"] = Y_
+    # labels are given we must check whether input is correct
+    if labels is not None:
+        if isinstance(label_palette, str):
+            colors = sns.color_palette(label_palette, n_components)
+            label_palette = dict(zip(np.unique(np.asarray(labels)), colors))
+        elif not isinstance(label_palette, dict):
+            msg = "When giving labels must supply palette in string or dictionary"
+            raise ValueError(msg)
+        if isinstance(cluster_palette, str):
+            colors = sns.color_palette(cluster_palette, n_components)
+            cluster_palette = dict(zip(np.unique(Y_), colors))
+        elif not isinstance(label_palette, dict):
+            msg = "When giving labels must supply palette in string or dictionary"
+            raise ValueError(msg)
+    else:
+        # no labels given we go to default.
+        colors = sns.color_palette(cluster_palette, n_components)
+        labels = np.unique(Y_)
+        cluster_palette = dict(zip(np.unique(labels), colors))
+        label_palette = dict(zip(np.unique(np.asarray(labels)), colors))
+
+    with sns.plotting_context(context=context, font_scale=font_scale):
+        dimensions = X.shape[1]
+        # we only want 1 scatter plot for 2 features
+        if X.shape[1] == 2:
+            dimensions = 1
+        pairplot, axes = plt.subplots(
+            dimensions, dimensions, figsize=figsize, squeeze=False
+        )
+        # this will allow for uniform iteration whether axes was 2d or 1d
+        axes = axes.flatten()
+        for i in range(dimensions):
+            for j in range(dimensions):
+                if i == j and X.shape[1] > 2:
+                    # take care of the distplot on diagonal
+                    for t, lab in zip([i for i in range(X.shape[1])], label_palette):
+                        sns.distplot(
```

Comment:
okay, talked to people on the team, can you bump this to `histplot` and also make our seaborn specifications `seaborn >= 0.11` as part of this PR? see #594

Reply:
so this would only be changing my distplot to histplot. or would you want me to change all distplot to histplot in plot.py?@bdpedigo

Reply:
do you mind changing the others too? forgot about that.

Reply:
@bdpedigo how would i go about changing the specification though?

Reply:
like would this be changing requirements.txt?

Reply:
https://github.com/microsoft/graspologic/pull/541/commits/edb473ee394a33c94aaac96cf771ed61472048fb